src/atoms/Element/index.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 /* eslint-disable import/first */
-import styles from './element.module.scss';
 import colors from 'atoms/Color/colors.scss';
 import spacing from 'atoms/Spacer/spacer.scss';
+import styles from './element.module.scss';
```

Comment:
can you double check that moving this didn't break anything? line 5 is makes it seem like the import order was done on purpose

Reply:
I don't even remember changing this or why - I changed it back.